modules/core/jvm/src/main/scala/gem/syntax/Stream.scala

```diff
@@ -49,7 +49,7 @@ final class StreamOps[F[_], O](val self: Stream[F, O]) {
             val aʹ = f(a, o)
             if (p(aʹ)) (matching :+ o, z) else (matching, aʹ)
           }).flatMap { case (matching, aʹ) =>
-            Pull.output(Chunk.vector(matching)) *> go(aʹ, tl)
+            Pull.outputChunk(Chunk.vector(matching)) >> go(aʹ, tl)
```

Comment:
There are some subtle strictness issues here that caused them to replace `*>` with `>>` … not worth worrying about but there is an explanation.